docs/details/image.dox

```diff
@@ -626,9 +626,13 @@ dimensions will stay the same as the original image's, but the rotated image's
 portions outside the dashed orange lines will be cropped, and the rest of the
 output image (the area between the solid black and solid orange lines) will be
 filled with zeros. However, if \p crop is false, then the output image's
-dimensions will be bigger (outermost box with dashed black lines) in order to
-accommodate all of the rotated image's data, and the remainder will be filled
-with zeros (the area between the solid orange lines and dashed black lines).
+dimensions will be bigger (at least in this illustration), as represented by
```

Comment:
I think `will be bigger (at least in this illustration)` should be something like the following.
```
...
dimensions might get bigger for some cases (as shown in the above illustration)
...
```